eng/pipelines/mono/templates/xplat-pipeline-job.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
     variables:
     - name: coreClrProductArtifactName
-      value: 'CoreCLRProduct_${{ parameters.runtimeVariant }}_$(osGroup)$(osSubgroup)_$(archType)_${{ parameters.liveRuntimeBuildConfig }}'
+      value: 'CoreCLRProduct__$(osGroup)$(osSubgroup)_$(archType)_${{ parameters.liveRuntimeBuildConfig }}'
```

Comment:
Is this now shared or why are we changing this?

Reply:
This was done to allow LLVM AOT Mono-specific jobs to use the one-and-only variant of CoreCLR that we build right now.

Support for CoreCLR `runtimeVariant`s was added in https://github.com/dotnet/runtime/pull/35841. But `runtimeVariant` isn't currently used at all when building CoreCLR. The choices I see right now are:
- remove `runtimeVariant` support entirely from eng/pipelines/coreclr/*.yml;
- build an `llvmaot` variant of CoreCLR, for later consumption by LLVM AOT Mono-specific jobs, that is identical to the non-variant CoreCLR; or
- explicitly depend on a shared non-variant CoreCLR.

Reply:
I think "explicitly depend on a shared non-variant CoreCLR." is the one that makes the most sense. I was mostly courious why was this change needed just to understand how the system worked.

It seems like we do have an extra `_` in between `CoreCLRProduct_` and `osGroup` though.

Reply:
> I think "explicitly depend on a shared non-variant CoreCLR." is the one that makes the most sense. I was mostly courious why was this change needed just to understand how the system worked.

Ah, got it.

> It seems like we do have an extra `_` in between `CoreCLRProduct_` and `osGroup` though.

Yeah. That's a consequence of how the product artifact name is concatenated: https://github.com/dotnet/runtime/pull/35841/files#diff-9d649abd6ea7b7db8a63e5dd8edfd14bR81

Reply:
I don't want to block this PR because of that since it seems like it is something we already heavily depend on and the change might be bigger that it seems.